--- a/debian/changelog
+++ b/debian/changelog
@@ -1,7 +1,7 @@
-wxsvg (1.0b10-lenny1) stable; urgency=low
+wxsvg (1.0b10-ubuntu1) hardy; urgency=low
 
   * packages for Debian Lenny, Sid and Ubuntu Hardy Heron
   * packages are split now in shared and devel libraries
   * invoked quilt and patch as dependencies
 
- -- Uwe Bugla <uwe.bugla@gmx.de>  Thu, 10 Jul 2008 20:27:00 +0200
+ -- Fabrice Coutadeur <coutadeurf@gmail.com>  Thu, 10 Jul 2008 20:27:00 +0200
--- a/debian/control
+++ b/debian/control
@@ -1,21 +1,21 @@
 Source: wxsvg
 Section: libs
 Priority: optional
-Maintainer: Uwe Bugla <uwe.bugla@gmx.de>
-Build-Depends: debhelper (>= 7.0.9), fakeroot (>= 1.9.5), automake (>= 1:1.10.1), libtool (>= 1.5.26), libwxgtk2.6-dev (>= 2.6.3.2.2), libwxbase2.6-dev (>= 2.6.3.2.2), libavformat-dev (>= 3:20080610-0.0), libavcodec-dev (>= 3:20080610-0.0), libswscale-dev (>= 3:20080610-0.0), libart-2.0-dev (>= 2.3.20), libpango1.0-dev (>= 1.20.2)
+Maintainer: Fabrice Coutadeur <coutadeurf@gmail.com>
+Build-Depends: quilt (>= 0.46-4), patch (>= 2.5.9-4), debhelper (>= 6.0.4ubuntu1), fakeroot (>= 1.9ubuntu1), automake (>= 1:1.10.1), libtool (>= 1.5.26-1ubuntu1), libwxgtk2.6-dev (>= 2.6.3.2.2-2ubuntu4), libwxbase2.6-dev (>= 2.6.3.2.2-2ubuntu4), libavformat-dev (>= 3:0.cvs20070307-5ubuntu7), libavcodec-dev (>= 3:0.cvs20070307-5ubuntu7), libswscale-dev (>= 3:0cvs20070307-5ubuntu7), libart-2.0-dev (>= 2.3.20), libpango1.0-dev (>= 1.20.1)
 
 Package: libwxsvg0
 Architecture: any
 Section: libs
-Depends: ${shlibs:Depends}, libwxgtk2.6-0 (>= 2.6.3.2.2), libwxbase2.6-0 (>= 2.6.3.2.2), libart-2.0-2 (>= 2.3.20), libpango1.0-0 (>= 1.20.2), ffmpeg (>= 3:20080610-0.0)
+Depends: ${shlibs:Depends}, libwxgtk2.6-0 (>= 2.6.3.2.2-2ubuntu4), libwxbase2.6-0 (>= 2.6.3.2.2-2ubuntu4), libart-2.0-2 (>= 2.3.20), libpango1.0-0 (>= 1.20.1), ffmpeg (>= 3:0.cvs20070307-5ubuntu7)
 Conflicts: wxsvg (<= 1.0b9)
 Description: A C++ library to create, manipulate and render SVG files
  This package contains the shared library.
  
 Package: libwxsvg-dev
 Architecture: any
 Section: libdevel
-Depends: libwxsvg0 (>= ${binary:Version}), libwxgtk2.6-dev (>= 2.6.3.2.2), libwxbase2.6-dev (>= 2.6.3.2.2), libavformat-dev (>= 3:20080610-0.0), libavcodec-dev (>= 3:20080610-0.0), libswscale-dev (>= 3:20080610-0.0), libart-2.0-dev (>= 2.3.20), libpango1.0-dev (>= 1.20.2)
+Depends: libwxsvg0 (>= ${binary:Version}), libwxgtk2.6-dev (>= 2.6.3.2.2-2ubuntu4), libwxbase2.6-dev (>= 2.6.3.2.2-2ubuntu4), libavformat-dev (>= 3:0.cvs20070307-5ubuntu7), libavcodec-dev (>= 3:0.cvs20070307-5ubuntu7), libswscale-dev (>= 3:0.cvs20070307-5ubuntu7), libart-2.0-dev (>= 2.3.20), libpango1.0-dev (>= 1.20.1)
 Conflicts: wxsvg (<= 1.0b9)
 Description: A C++ library to create, manipulate and render SVG files
  This package contains the development files.
